build.sbt

```diff
@@ -15,6 +15,11 @@ libraryDependencies += "com.databricks" % "dbutils-api_2.12" % "0.0.5" % Provide
 libraryDependencies += "com.amazonaws" % "aws-java-sdk-s3" % "1.11.595" % Provided
 libraryDependencies += "io.delta" % "delta-core_2.12" % "1.0.0" % Provided
 libraryDependencies += "org.scalaj" %% "scalaj-http" % "2.4.2"
+//For API Modernization
+libraryDependencies += "com.tdunning" % "json" % "1.8"
```

Comment:
see comments throughout code. I'm very sensitive to adding new dependencies but am happy to do so where justified. We just need to make sure this is what we want to use as a team.

Reply:
removed the libs